aws/service.go

```diff
@@ -700,6 +701,27 @@ func KinesisService(ctx context.Context, d *plugin.QueryData, region string) (*k
 	return svc, nil
 }
 
+// KinesisAnalyticsV2Service returns the service connection for AWS Kinesis Analytics V2 service
```

Comment:
Is `Kinesis Analytics V2` is a separate word, or it should be `Kinesis Analyticsv2`?